geode-protobuf/src/main/proto/clientProtocol.proto

```diff
@@ -66,7 +66,7 @@ message Response {
         RemoveAllResponse removeAllResponse = 7;
         ListKeysResponse listKeysResponse = 8;
 
-        ErrorResponse errorResponse = 13;
+        Error error = 13;
```

Comment:
To keep the "Response -> OperationalResponse" hierarchy, I'd prefer to keep "ErrorResponse" that contains an "Error" object.
We _could_ inline it, but it detracts from the modus-operandi